README.md

```diff
@@ -37,13 +37,24 @@ submodules i.e. `git clone https://github.com/nhsuk/gp-finder.git && cd
 gp-finder/ && git submodule update --init --recursive`
```

Comment:
Not part of your changes, but there's a typo at line 30, says 'scrips' instead of 'scripts'